torch/fx/interpreter.py

```diff
@@ -153,6 +153,7 @@ def run(self, *args, initial_env : Optional[Dict[Node, Any]] = None, enable_io_p
 
     @contextmanager
     def _set_current_node(self, node):
+        self.current_node = node
```

Comment:
why?

Reply:
ChangeInputOutputSignature uses self.current_node. 

Alos, having self.current_node available is generally useful.

Reply:
Nit, kinda unrelated to this PR. sounds fine otherwise!